### PR TITLE
feat: added angular structure to the project

### PR DESCRIPTION
**Pull request by robpan38**

### CODE REVIEW

**Comment by anbalase** [comment]
I see a merge commit in the history. 
nit: Also, use present tence in the commit msg pls.

File: src/app/chat/app-container/chat-container/chat-container.component.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<div class="chat-header">
+    <p class="chat-name">#Chat name placeholder</p>
+</div>
+<div class="messages-container">
+    <div class="message-container"></div>
```
Comment (anbalase) [inline src/app/chat/app-container/chat-container/chat-container.component.html:5]: how 'bout a ngFor?

File: src/app/chat/app-container/chat-container/chat-container.component.html [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    <div class="message-container"></div>
+    <div class="message-container"></div>
+    <div class="message-container"></div>
+    <div class="message-container"></div>
+    <div class="message-container"></div>
+</div>
+<div class="chat-input-container">
+    <textarea class="chat-input"></textarea>
+    <button class="chat-send-btn">Send</button>
```
Comment (anbalase) [inline src/app/chat/app-container/chat-container/chat-container.component.html:16]: pls use `<form>` where needed

File: src/app/chat/app-container/chat-container/chat-container.component.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-chat-container',
+  templateUrl: './chat-container.component.html',
+  styleUrls: ['./chat-container.component.scss']
```
Comment (anbalase) [inline src/app/chat/app-container/chat-container/chat-container.component.ts:6]: use OnPush https://www.thinktecture.com/en/angular/whats-the-hype-onpush/#:~:text=The%20OnPush%20strategy%20changes%20Angular's,a%20subtree%20for%20that%20component.

File: src/app/chat/app-container/chat-container/chat-container.component.ts [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+})
+export class ChatContainerComponent implements OnInit {
+
+  constructor() { }
```
Comment (anbalase) [inline src/app/chat/app-container/chat-container/chat-container.component.ts:10]: don't leave in empty methods or constructors

File: src/app/chat/app-container/users-container/users-container.component.html [hunk 5]
```diff
@@ -0,0 +1,27 @@
+<div class="user-container">
+    <div class="user-pic">
+        <img src="../assets/user_pics/cat-pic.jpg" class="user-pic-img" />
+    </div>
+    <p class="user-name">Vali Vijelieeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee</p>
```
Comment (anbalase) [inline src/app/chat/app-container/users-container/users-container.component.html:5]: se stie!

File: src/app/login/login-input-container/login-input-container.component.html [hunk 6]
```diff
@@ -0,0 +1,7 @@
+<div class="login-input-container">
+    <input class="login-input-field" placeholder="Enter username here..."
```
Comment (anbalase) [inline src/app/login/login-input-container/login-input-container.component.html:2]: forms pls

**Review by mihneat — APPROVED** [review]
the feedback will be implemented on a different branch, deriving from main, after the merge

**Pull request merged**